src/app/modules/group/group-routing.module.ts

```diff
@@ -18,6 +18,9 @@ const routes: Routes = [
     children: [
```

Comment:
I found an unrelated bug:
When using the update-profile-test user and updating its data via the modify button, the section above the tabs is not refreshed. (I edited the website and the first name.)
I'll create an issue for that.
![image](https://user-images.githubusercontent.com/16257159/170997843-3dfcde96-98db-43cd-9407-a4a6c2327744.png)